configs/intuit_auto.json

```diff
@@ -1,24 +1,55 @@
 {
   "index_name": "intuit_auto",
   "start_urls": [
-    "https://intuit.github.io/auto/"
+    {
+      "url": "https://intuit.github.io/auto/",
+      "page_rank": 5,
+      "tags": ["docs"]
+    },
```

Comment:
Should be the last item as the first matching prevail

Reply:
fixed1